src/screens/Followings/FollowersScreen.js

```diff
@@ -103,7 +114,8 @@
 
       <Followings
         isLoading={isLoading}
-        dataFollower={dataFollower}
+        dataFollower={followedUsers}
+        dataUnfollowed={unfollowedUsers}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new props `dataFollower` and `dataUnfollowed` are passed to the `Followings` component. Make sure that the `Followings` component is updated to handle these new props correctly.
```diff
+ dataFollower={followedUsers}
+ dataUnfollowed={unfollowedUsers}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->